index.html

```diff
@@ -9,48 +9,48 @@
 </head>
 
 <body>
-    <h1 style = font-size:0px> Reliable, efficient delivery</h1>
+    <h1>Realiable, efficient delivery </h1>
     <h1 >Powered by Technology</h1>
-    <h1 style = font-size:20px>Our Artificial Intelligence powered tools use millions of project data points to ensure that your project is successful</h1>
+    <h1 style = font-size:20px style = text-align:center>Our Artificial Intelligence powered tools use millions of project data points to ensure that your project is successful</h1>
     
     <div class = "row">
-        <div class = "column">
-            <div class="boxStyle0 supervisor">
+         <!-- <div class = "column0">  -->
+            <div class="boxStyle0 supervisor" >
                 <h1 class = "boxHeader">Supervisor</h1>  
                 <h2>Monitors activity to identify project roadblocks</h2>
                 <img src="images/icon-supervisor.svg" alt="supervisorPic">
             </div>
-        </div>
+         <!-- </div>  -->

```

Comment:
Removed the column class, since it was causing issues with the formatting of the flex items